toolchain/lower/file_context.cpp

```diff
@@ -108,9 +108,9 @@ auto FileContext::LowerDefinitions() -> void {
       // constant unless the variable is unnamed, in which case we need to
       // create it now.
       llvm::GlobalVariable* llvm_var = nullptr;
-      if (sem_ir().constant_values().Get(inst_id).is_constant()) {
-        llvm_var = cast<llvm::GlobalVariable>(
-            GetGlobal(inst_id, SemIR::SpecificId::None));
+      auto const_id = sem_ir().constant_values().Get(inst_id);
+      if (const_id.is_constant()) {
```

Comment:
Very minor scoping suggestion

```suggestion
      if (auto const_id = sem_ir().constant_values().Get(inst_id); const_id.is_constant()) {
```